src/reducers/index.js

```diff
@@ -96,6 +100,16 @@ const combinedReducer = combineReducers({
     TOGGLE_SHOW_FEATURE_HINTS,
     'showFeatureHints'
   ),
+  showDatasetPreviews: (state = true, action) => {
```

Comment:
can we have the block inside its own reducer as before ? This is more of a reducer block and best kept separate, instead of keeping it inside combineReducers to be consistent. Let me know if this was needed or if I am missing something here.